src/services/user.tsx

```diff
@@ -20,13 +21,12 @@ import { base_url } from "../env";
 class User {
   constructor(
     public userName: string,
-    public email: string,
     public userId: string
   ) {}
 
   static fromDocument(doc: DocumentData): User {
-    const { userName, email, userId } = doc;
```

Comment:
Why are we getting rid of email from the user class?

Reply:
Users are public in our database, and so we need to not store any sensitive information in them. They must be public because we need to allow the backend to query them when a user is first signing up to make sure they don't duplicate a username

Reply:
Ohhh I see, that makes sense